openlibrary/templates/lists/list_overview.html

```diff
@@ -2,7 +2,8 @@
 
 $ remove = (list.owner.key == user_key)
 $ actionable = 'class=actionable-item' if remove else ''
-<li $actionable>
+$ data_list_key = 'data-list-key=%s' % list.key if actionable else ''
+<li $actionable $data_list_key>
```

Comment:
Hey @jimchamp do you mind if I commit this change? I think that should let both our PRs co-exist on testing (mine is https://github.com/internetarchive/openlibrary/pull/8168 )

```suggestion
<li $actionable $:cond(actionable, 'data-list-key=%s' % list.key)>
```

Reply:
Messaged you on slack and you preferred to remove it from testing. Removed it :+1: Accept this suggestion at your discretion 😋